### ISSUE TITLE
Popup

### ISSUE THREAD

**Issue by Oksana13** [naveenkumarr05/weather-app#18]
A bit strange realisation for popup.. Usually it's done through a variable which is set to `true` by click for open it and `false` by closing

https://github.com/naveenkumarr05/weather-app/blob/89bbedf3d4fd693a7f39c6db777a1ad077478127/src/components/WeatherReport.vue#L5

**Comment by naveenkumarr05** [comment]
yeah correct only but I will change it, here I have used complete css styles for it
